layouts/pay/section.html

```diff
@@ -200,13 +122,13 @@ <h2 class="f4 fw6 tl">Thank you for installing a mesh node!</h2>
 
     var payButton = document.getElementById('payButton');
     payButton.addEventListener('click', function(e) {
-
-      if (parseInt(donationAmount) && donationAmount > 0) {
+      amount = document.querySelector('#customAmount').textContent;
```

Comment:
You don't need to set the amount onfocus because we're keeping he description block the same if it's edited. We just pull it once when they click pay to set the amount